IBPSA/Resources/C-Sources/getTimeSpan.c

```diff
@@ -0,0 +1,133 @@
+/*
+ * getTimeSpan.c
+ *
+ *  Created on: Apr 16, 2019
+ *      Author: jianjun
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/stat.h>
+
+#include "getTimeSpan.h"
+
+/*
+ * Function: concat
+ * -----------------
+ *  Concatenate two strings. This function calls malloc and hence
+ *  the caller must call free when the returned string is no longer used.
+ *
+ *  s1: string one
+ *  s2: string two
+ *
+ *  returns: concatenate strings (s1 + s2)
+ */
+char *concat(const char *s1, const char *s2) {
+  const size_t len1 = strlen(s1);
+  const size_t len2 = strlen(s2);
+  char *result = malloc(len1 + len2 + 1);
+  if (result == NULL){
+    ModelicaError("Failed to allocate memory in getTimeSpan.c");
+  }
+  strcpy(result, s1);
+  strcat(result, s2);
+  return result;
+}
+
+/*
+ * Funciton: getTimeSpan
+ * ---------------------
+ * Get start and end time of weather data.
+ *
+ * filename: weather data file path
+ * tabNam: name of table on weather file
+ * timeSpan: vector [start time, end time]
+ */
+int getTimeSpan(const char * filename, const char * tabNam, double* timeSpan) {
+  double firstTimeStamp, lastTimeStamp, interval;
+  int rowCount, colonCount;
```

Comment:
doesn't this have to be 'columnCount'?
Similarly, all other 'colon' variables have to renamed?